content/en/docs/setup/getting-started/index.md

```diff
@@ -48,10 +48,10 @@ Follow these steps to get started with Istio:
     {{< /tip >}}
 
 1.  Move to the Istio package directory. For example, if the package is
-    `istio-{{< istio_full_version >}}`:
+    `istio-1.7.0`:
 
     {{< text bash >}}
-    $ cd istio-{{< istio_full_version >}}
+    $ cd istio-1.7.0
```

Comment:
I think it would be better to leave this as it was and change line 53 to:
```
{{< text syntax=bash snip_id=none >}}
```
and then add the following just before [line 107 in snip.py](https://github.com/istio/istio.io/blob/8b5cf440ce0402814c31371eb5a60f9e24620456/scripts/snip.py#L107):
```
if match.group(1) == none:
    continue
```
```

Reply:
Fixed.